Modules/Core/Common/test/itkFixedArrayGTest.cxx

```diff
@@ -324,3 +325,22 @@ TEST(FixedArray, StdMemberFunctionsWork)
   d3arr.data()[0] = 10;
   EXPECT_EQ(cdata[0], 10);
 }
+
+
+// Tests that each element of a value-initialized FixedArray is itself value-initialized. By definition,
+// "value-initialization" is performed when an object is constructed with an empty {} initializer.
+TEST(FixedArray, ValueInitialized)
+{
+  const auto expectEachElementValueInitialized = [](const auto & fixedArray) {
+    for (const auto & element : fixedArray)
+    {
+      using ValueType = std::remove_reference_t<decltype(element)>;
+
+      EXPECT_TRUE(element == ValueType{});
+    }
+  };
+
+  expectEachElementValueInitialized(itk::FixedArray<int, 3>{});
```

Comment:
From here, pass 0, 0.0f, and nullptr as expectedValue.

Reply:
Sorry but I believe it's beyond the scope of this test to check that `T{}` equals 0 when T is an int, and that `T{}` equals `nullptr` when T is a pointer. That's a fundamental property of C++.

We could write such tests, like `EXPECT_EQ(int{}, 0)` and  `EXPECT_EQ(float{}, 0.0f)`, but I don't think they belong in a FixedArray test. They would fit better in a fundamental C++ feature test.